### PR TITLE
Assignment 4 testing code

### PR DESCRIPTION
**Pull request by gpsingh12**
task_03 
If i am choosing Prequalified as Yes.
Amount is calculated using the value 3
If I use Prequalified as No
Amount is calculated using the same value , it should calculate it using 4.
If we go to the next loop Duration 16 -20 samething happens.

Also I am using If else in two ways, 
1. PRINCIPAL and DURATION together,  Prequalified in next loop

2. All in the same if statement.

Please advice which one is better. 
Thanks

### CODE REVIEW

File: task_03.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import decimal
+import fractions
+NAME = raw_input('What is your name? ')
+
+PRINCIPAL = raw_input('What is amount of your Principal? $ ')
+PRINCIPAL = int(PRINCIPAL)
+DURATION = raw_input('for how many years is this loan being borrowed? ')
+DURATION = int(DURATION)
+Prequalified = raw_input('Are you pre-qualified for this loan? ')
+
```
Comment (cheuschober) [inline task_03.py]: You can make your tests a bit easier to manage if you test upfront, for example::

    COLOR = raw_input('Fav. color?' )[:1].lower()
    LIKES_GREEN = COLOR == 'c'

Here, I take my input and slice it to 1 character and lower it to 'normalize' the data. Then I use a simple expression to save the result of the test. Now I can use that in my if's as:

    if LIKES_GREEN:
        print 'Yay Green!'
    else:
        print 'Boo!'

Comment (gpsingh12) [inline task_03.py]: Thanks Prof.

File: task_03.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+if (PRINCIPAL <= 199999) and (DURATION <= 15) and (Prequalified == 'Yes' or 'y'):
```
Comment (cheuschober) [inline task_03.py]: Thought it's not required, you could try nesting your conditions to see if that works.

File: task_02.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" Docstring. """
+
+DAY = raw_input('What day is it? ')
+WEEKDAY = DAY == 'Sat' or DAY == 'Sun' or DAY == ' Saturday' or DAY == 'Sunday'
+TIME = raw_input('What time is it, enter 4 digits format? ')
+TIME = int(TIME)
+DISPLAY_TIME = TIME < '0600'
```
Comment (cheuschober) [inline task_02.py]: Since '0600' as you have it here is a *string* your comparison was always going to fail.

File: task_03.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+DURATION = int(DURATION)
+DUR = (DURATION <= 15)
+DRN = (16 <= DURATION <= 20)
+DRT = (21 <= DURATION <= 30)
+Pre_qualified = raw_input('Are you pre-qualified for this loan? ')
+PRE_Q = Pre-qualified == 'Yes' or Pre-qualified == 'y'
+PRE_QL = Pre-qualified == 'No' or 'n'
+
+if PRN and DUR and PRE_Q:
+    Interest Rate = 3.63
```
Comment (cheuschober) [inline task_03.py]: Your interest rates here are floats, not decimals. The problem asked for decimal level precision which is a major contributing factor to your success in this case.

**Pull request merged**